src/ensembl/src/content/app/species-selector/state/speciesSelectorActions.ts

```diff
@@ -187,12 +177,9 @@ export const ensureSpeciesIsEnabled: ActionCreator<ThunkAction<
   dispatch(toggleSpeciesUseAndSave(genomeId));
 };
 
-export const fetchPopularSpecies: ActionCreator<ThunkAction<
-  void,
-  any,
-  null,
-  Action<string>
->> = () => async (dispatch) => {
+export const fetchPopularSpecies: ActionCreator<
```

Comment:
Here and in other places, is it worth getting rid of `ActionCreator` in this PR itself ?

Reply:
Done for this actions file.

Curiously, the redux team [has updated](https://github.com/reduxjs/redux/blame/master/src/types/actions.ts#L52) type definitions for the ActionCreator, which might have made it safer.

Reply:
Cool, I guess we don't have to worry about removing it going forward?

Reply:
It's great if it no longer does harm. I guess we don't need to worry much about it, but we can justify the further gradual removal of this type by the fact that, for us, it doesn't seem to add anything of value.